tests/service_named_result/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2020 Lucio Franco
```

Comment:
I don't think we need a license here?

Reply:
I've continued the existing pattern of including license files as followed by other examples. Should we stop including this file?

Reply:
ah yeah this is fine, we should be soft symlinking them but its ok since you're right the other crates in the test folder has the same thing.